driver-lambda/template.yaml

```diff
@@ -0,0 +1,69 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Java driver lambda function test
+Parameters:
+  MongoDbUri:
+    Type: String
+    Description: The MongoDB connection string.
+
+Globals:
+  Function:
+    Timeout: 30
+    MemorySize: 128
+    Tracing: Active
+  Api:
+    TracingEnabled: true
```

Comment:
Do we need tracing enabled for this test?

Reply:
I believe this is from [the spec](https://github.com/mongodb/specifications/blob/master/source/faas-automated-testing/faas-automated-testing.rst#project-initialization) saying:
> Drivers MAY choose to also enable X-Ray tracing and CloudWatch Application Insights during these next steps.
My thought was that this was suggested because it might be needed in the future or come in handy.

Reply:
Thanks for mentioning the spec's suggestion about enabling AWS X-Ray tracing. Since we don't seem to use distributed tracing for Lambda invocation time testing, I would suggest not using it until required. AWS X-Ray might introduce extra charges

Reply:
I have disabled this